ds-caselaw-ingester/tests.py

```diff
@@ -799,17 +799,17 @@ def test_fcl_ingest_no_email(self, bulk, new, updated, fcl_ingest):
         bulk.assert_not_called()
 
     @patch("lambda_function.Metadata.force_publish", new_callable=PropertyMock)
-    def test_s3_ingest_no_email_if_publish(self, property, bulk, new, updated, s3_ingest):
-        property.return_value = True
+    def test_s3_ingest_no_email_if_publish(self, prop, bulk, new, updated, s3_ingest):
+        prop.return_value = True
```

Comment:
Not sure I love this one -- yes, it's shadowing @property, but `prop` is not a very descriptive name. Can we go for `mock_property` or similar.